Dockerfile

```diff
@@ -78,23 +78,22 @@ RUN set -ex \
     && go env -w GO111MODULE=auto \
     && go build -v -i -o /app/console_node $GOPATH/src/console_node
 
+### Final Stage ###
+# Start with a fresh image so build tools are not included
+#
 # NOTE:
 #  We need to switch to the below image, but for now it does not include the 'nobody' user
```

Comment:
This note should be updated - we are switching to the image, don't need to do it.

Reply:
Fixed.